bulbs/poll/models.py

```diff
@@ -126,7 +126,12 @@ def get_sodahead_data(self):
         response = requests.get(SODAHEAD_POLL_ENDPOINT.format(self.sodahead_id))
 
         if not response.ok:
-            raise SodaheadResponseError(response.text)
+            return {
```

Comment:
Are there specific response codes that signify an "existing but unpublished" poll? If so, could filter on that else raise error.